src/repo/repository.rs

```diff
@@ -116,6 +117,13 @@ pub struct ExtractResult {
 pub struct Repository {
     /// The path containing the [`Repository::data_dir`] directory.
     path: PathBuf,
+    /// With the addition of `elfshaker clone`, there are now longer running tasks
+    /// for which it would be useful to provide the user with a progress bar.
```

Comment:
I think this comment should not make reference to contemporary events. I think you can just drop these two lines without loss of useful information.

Reply:
Agreed, thanks.